manila/share/manager.py

```diff
@@ -135,6 +135,14 @@
                 default=False,
                 help='Offload pending share ensure during '
                      'share service startup'),
+    cfg.IntOpt('force_delete_time',
+               default=2,
```

Comment:
Do you think it makes sense to be able to disable the feature?
E.g. if force_delete_time is set to zero (0) -> then there is no force deletion performed?

Reply:
added 0x0 value for disable